packages/react/src/components/DetailsList/DetailsColumn.base.tsx

```diff
@@ -119,6 +119,9 @@ export class DetailsColumnBase extends React.Component<IDetailsColumnProps> {
           {...(!hasInnerButton && accNameDescription)}
           aria-sort={column.isSorted ? (column.isSortedDescending ? 'descending' : 'ascending') : 'none'}
           aria-colindex={columnIndex}
+          data-is-focusable={
+            !hasInnerButton && column.columnActionsMode !== ColumnActionsMode.disabled ? 'true' : undefined
```

Comment:
Can you add a comment (maybe both here and below) explaining why it's set up this way?

Reply:
will do!